packages/core/src/app/order/MandateTextComponent.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FunctionComponent } from 'react';
+import { TranslatedString } from '@bigcommerce/checkout/locale';
+
+export interface MandateTextComponentProps {
+    mandateText: {
+        [key: string]: string;
+    },
+    methodId: string;
+    providerId: string;
+}
+
+export const MandateTextComponent: FunctionComponent<MandateTextComponentProps> = ({
+   mandateText,
+   providerId,
+   methodId,
+}) => {
+    if (!Object.keys(mandateText).length) {
+        return null;
+    }
```

Comment:
May I ask you to add tests for this flow as well?

Reply:
added test